src/lib/molecules/Charts/HeatMapChartD3/HeatMapChartD3.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { MouseEvent, useEffect, useMemo, useRef, useState } from 'react';
+import Highcharts from 'highcharts';
+import HS_map from 'highcharts/modules/treemap';
+import Legend, { LegendAppearances } from './Legend';
+
+import * as d3 from 'd3';
+
+// Components
+import Empty from '../../../atoms/Empty';
+import BusyLoader from '../../../atoms/BusyLoader';
+
+// Styles
+import './HeatMapChartD3.scss';
+import Tooltip, { ITooltipProps } from './Tooltip';
```

Comment:
Pease put this import under Components , and ITooltipProps under Types

Reply:
Done